md/dev/mediquette.txt

```diff
@@ -0,0 +1,74 @@
+
+Etiquette @medic
+
+Be kind to the plants and animals.
```

Comment:
I would add "Try your hardest not to be an asshole." That to me is a requirement for working with people. That cuts short to any aggressive or passive-aggressive behavior.